transcom/mymove
# B 20473 int2

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A980089&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

Update to verbiage on cancel Confirmation modal per Dan S.
Also, fixing a random space change that somehow got in on my first INT pr.

### How to test

1. Access the SC Counseling queue.  Select a move in Needs Services Counseling status.
2. Click on the Cancel move link on the top of the Move details page.
3. Verify the text reads as this:
"If you proceed with cancellation and later want to process a move request for these orders, you will have to start again with a new move."
